memcached/README.md

```diff
@@ -0,0 +1,45 @@
+# YCSB Memcached binding
+
+## Quick Start
+
+This section describes how to run YCSB on memcached.
+
+### 1. Install and start memcached service on the host(s)
+
+### 2. Install Java and Maven
+
```

Comment:
Done for step 1. For step 2, it seems like this is binding-agnostic, so presumably it should be documented in a single place, and every binding can just point to it.

In fact, even step 5 is the same for everything, just substituting the binding is sufficient.

What are your thoughts about moving the "install Java / Maven" to a common place and linking to it? If so, where should it be located: top-level README, perhaps?